sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -525,23 +403,18 @@ class SQLQueryTestSuite extends QueryTest with SharedSparkSession with SQLHelper
         AnsiTestCase(testCaseName, absPath, resultFile) :: Nil
       } else if (file.getAbsolutePath.startsWith(s"$inputFilePath${File.separator}timestampNTZ")) {
         TimestampNTZTestCase(testCaseName, absPath, resultFile) :: Nil
+      } else if (analyzerTestCaseList.contains(file.getName.toLowerCase(Locale.ROOT))) {
```

Comment:
I think analyzerTestCaseList should cover the `ansi`/`TimestampNTZ`/`postgreSQL` input queries as well

Reply:
Good idea, done!